tests/installation/install_and_reboot.pm

```diff
@@ -61,6 +61,9 @@ sub run() {
     if (check_var('ARCH', 's390x')) {
         push(@tags, 'additional-packages');
     }
+    # detect "Wrong Digest" error to end test earlier
+    push(@tags, 'yast2_wrong_digest');
```

Comment:
if you unconditionally 'add' value just do it in line 45 where tags array created